shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-core-spring/shardingsphere-jdbc-core-spring-namespace/src/main/resources/META-INF/namespace/datasource.xsd

```diff
@@ -29,7 +29,7 @@
                 <xsd:element ref="beans:props" minOccurs="0" />
             </xsd:all>
             <xsd:attribute name="id" type="xsd:string" use="required" />
-            <xsd:attribute name="schema-name" type="xsd:string" />
+            <xsd:attribute name="schema-name" type="xsd:string" default="logic_db"/>
```

Comment:
Please keep one space between `"logic_db"` and `/>`, just keep consist with others

Reply:
ok, i will done